src/aleph/vm/conf.py

```diff
@@ -336,6 +342,12 @@ def check(self):
                 int(ipv4_pool_length) <= settings.IPV4_NETWORK_PREFIX_LENGTH
             ), "The IPv4 address pool prefix must be shorter than an individual VM network prefix"
 
+        if self.USE_CONFIDENTIAL_COMPUTING:
```

Comment:
confidential require qemu so we should check that the qemu support is also properly enabled